packages/frontend/src/components/ProjectConnection/WarehouseForms/RedshiftForm.tsx

```diff
@@ -70,183 +77,278 @@ const RedshiftForm: FC<{
-                                : `Generate public key`
-                        }
-                        onClick={() => mutate()}
-                        loading={isLoading}
-                        disabled={disabled || isLoading}
-                    />
+                        <Controller
+                            name="warehouse.sslmode"
+                            defaultValue="prefer"
+                            render={({ field }) => (
+                                <Select
+                                    name={field.name}
+                                    label="SSL mode"
+                                    description={
+                                        <p>
+                                            This controls how dbt connects to
+                                            Redshift databases using SSL. You
+                                            can see more details in{' '}
+                                            <Anchor
+                                                target="_blank"
+                                                href="https://docs.getdbt.com/docs/core/connect-data-platform/redshift-setup#sslmode-change"
+                                                rel="noreferrer"
+                                            >
+                                                dbt documentation
+                                            </Anchor>
+                                            .
+                                        </p>
+                                    }
+                                    data={[
+                                        'disable',
+                                        'no-verify',
+                                        'allow',
+                                        'prefer',
+                                        'require',
+                                        'verify-ca',
+                                        'verify-full',
+                                    ].map((x) => ({ value: x, label: x }))}
+                                    value={field.value}
+                                    onChange={field.onChange}
+                                    disabled={disabled}
+                                />
+                            )}
+                        />
+                        <Controller
+                            name="warehouse.ra3Node"
+                            render={({ field }) => (
+                                <Switch.Group
+                                    label="Use RA3 node"
+                                    description="Allow dbt to use cross-database-resources."
+                                    value={field.value ? ['true'] : []}
+                                    onChange={(values) =>
+                                        field.onChange(values.length > 0)
+                                    }
+                                    size="md"
```

Comment:
Hi @ZeRego , do we need `size="md"` here?

Reply:
We don't need it but I felt it looked slightly better.
With
<img width="423" alt="Screenshot 2023-09-21 at 14 56 15" src="https://github.com/lightdash/lightdash/assets/9117144/c633124b-6e1c-4cf5-865e-565abae72585">
Without
<img width="430" alt="Screenshot 2023-09-21 at 14 56 02" src="https://github.com/lightdash/lightdash/assets/9117144/6e8cd7a4-3974-4c95-b5af-4ebf148f9988">